### PR TITLE
Log mostly at the debug lebel; use template strings

### PR DESCRIPTION
**Pull request by jamieparkinson**

### CODE REVIEW

File: packages/apps/api-authorizer/src/index.ts [hunk 1]
```diff
@@ -89,18 +81,11 @@ export async function lambdaHandler(
...
-        ' ' +
-        event.resource +
-        '] with path parameters [' +
-        event.pathParameters +
-        ']'
+    console.debug(
+      `Access token [${accessToken}] for user [${JSON.stringify(
+        auth0UserInfo
+      )}]` +
+        `cannot operate on [${event.httpMethod} ${event.resource}] with path parameters [${event.pathParameters}]`
```
Comment (alexwlchan) [inline packages/apps/api-authorizer/src/index.ts:88]: Probably not one to fix now, but we should look at exactly what's in that `auth0UserInfo` variable before we go logging it everywhere.

File: packages/apps/api/src/utils/email.ts [hunk 2]
```diff
@@ -113,18 +113,13 @@ export default class EmailClient {
...
       })
       .catch((error) => {
-        console.log(
-          'An error occurred sending SMTP email [' +
-            smtpParams +
-            ']: [' +
-            error +
-            ']'
+        console.error(
+          `An error occurred sending SMTP email [${smtpParams}]`,
```
Comment (alexwlchan) [inline packages/apps/api/src/utils/email.ts:121]: Ditto `smtpParams` and possible user email addresses.

File: packages/shared/auth0-client/src/auth0.ts [hunk 3]
```diff
@@ -84,12 +84,11 @@ function toAuth0Profiles(users: any[]): Auth0Profile[] {
...
     } catch (e) {
-      console.log(
-        'An error occurred converting Auth0 user [' +
-          JSON.stringify(user) +
-          '] to Auth0Profile: [' +
-          e +
-          ']'
+      console.error(
+        `An error occurred converting Auth0 user [${JSON.stringify(
+          user
```
Comment (alexwlchan) [inline packages/shared/auth0-client/src/auth0.ts:89]: Ditto.

**Review by alexwlchan — APPROVED** [review]

File: packages/apps/api-authorizer/src/index.ts (hunk 1 above)
Comment (jamieparkinson) [reply packages/apps/api-authorizer/src/index.ts:88]: shouldn't be an issue at the debug level, but yes, should be done especially for the other cases

Comment (kenoir) [reply packages/apps/api-authorizer/src/index.ts:88]: Definitely PII: https://github.com/wellcomecollection/identity/blob/45c9badfc2aa8ba7197530b1a64193836a0056fb/packages/shared/auth0-client/src/auth0.ts#L151

**Review by kenoir — COMMENTED** [review]
Good points about logging PII raised here.

File: packages/apps/api-authorizer/src/index.ts (hunk 1 above)
Comment (alexwlchan) [reply packages/apps/api-authorizer/src/index.ts:88]: > shouldn't be an issue at the debug level

Not sure I agree. I don't love the idea that changing the log level suddenly starts putting PII into our logging cluster.

**Comment by alexwlchan** [comment]
I've created a ticket to track the PII discussions: https://github.com/wellcomecollection/platform/issues/5166

File: packages/apps/api-authorizer/src/index.ts (hunk 1 above)
Comment (jamieparkinson) [reply packages/apps/api-authorizer/src/index.ts:88]: True (although I got the [idea](https://wellcome.slack.com/archives/CUA669WHH/p1618396869015100?thread_ts=1618395149.009100&cid=CUA669WHH) from you 😬) - I will return to this in my next PR

Comment (alexwlchan) [reply packages/apps/api-authorizer/src/index.ts:88]: The person who wrote that message looks very shifty. I wouldn't trust anything they say! 😉

Comment (jamieparkinson) [reply packages/apps/api-authorizer/src/index.ts:88]: Encouraging accidental leaking of PII is the kind of thing a spy might do

**Pull request merged**